CMakeLists.txt

```diff
@@ -1148,11 +1148,14 @@ if(HPX_WITH_NETWORKING)
     CATEGORY "Parcelport"
   )
   if(HPX_WITH_PARCELPORT_GASNET)
+    if(...is cray aries available)
```

Comment:
I don't think CMake will like this. I was using just some kind of pseudo code in my suggestion - sorry.